### ISSUE TITLE
VOC Dataset issue

### ISSUE THREAD

**Issue by ANSHUMAN87** [qfgaohao/pytorch-ssd#59]
Hi I downloaded VOC dataset from http://host.robots.ox.ac.uk:8080/eval/downloads/   post signin.
But when i executed the eval_ssd as below snapshot, i got the error.

![image](https://user-images.githubusercontent.com/32511895/61102277-d35c6280-a48a-11e9-8a46-3a91af78ecc7.png)

Please help me to resolve this problem, Thanks!

**Comment by blacksailer** [comment]
Is there file ``2008_000001.xml`` in ``VOC2012/Annotations``?

**Comment by ANSHUMAN87** [comment]
No, the file is not present, similarly many others also. That's the error about. Please help, thanks.

**Comment by mekong0404** [comment]
@ANSHUMAN87 its a path problem, with file extension, wither u need to fix the file extension in the code or remove it fromt he file. try printing the annotation_file

**Comment by MLbeginner2** [comment]
Have you solved this problem？

**Comment by ANSHUMAN87** [comment]
@MLbeginner2 : No i have not solved it yet, @mekong0404 : This is not path issue, the files are missing as per the names mentioned in the files. If need to be removed manually, that i am aware, and i have done few in that way, but that is not right. As the dataset  should be free of errors.

**Comment by MLbeginner2** [comment]
> @MLbeginner2 : No i have not solved it yet, @mekong0404 : This is not path issue, the files are missing as per the names mentioned in the files. If need to be removed manually, that i am aware, and i have done few in that way, but that is not right. As the dataset should be free of errors.

Since it is used for verification, I suggest you try it with a val set.
